Mlem/Views/Shared/Composer/PostComposerView.swift

```diff
@@ -261,4 +278,36 @@ struct PostComposerView: View {
         .navigationBarColor()
         .navigationBarTitleDisplayMode(.inline)
     }
+    
+    @ViewBuilder
+    var headerView: some View {
+        HStack {
+            CommunityLabelView(
+                community: editModel.community,
+                serverInstanceLocation: .bottom,
+                overrideShowAvatar: true
+            )
+            Spacer()
+            if let person = siteInformation.myUserInfo?.localUserView.person {
+                UserLabelView(
+                    person: person,
+                    serverInstanceLocation: .bottom,
+                    overrideShowAvatar: true
+                )
+                .environment(\.layoutDirection, layoutDirection == .leftToRight ? .rightToLeft : .leftToRight)
+            }
+        }
+        .overlay {
+            if let slurMatch = titleSlurMatch == nil ? bodySlurMatch : titleSlurMatch {
+                ZStack {
+                    Capsule()
+                        .fill(.red)
+                    Text("\"\(slurMatch)\" is disallowed.")
```

Comment:
Maybe we even want to centralize this into a struct? It seems a little silly for a single line of text but it would guarantee consistency if we ever change it down the line

Reply:
Hmm, I'd rather use a static property of `AppConstants` like we do for some destructive actions if we want to keep the text the same. If we every get around to localisation, we can just use the same localisation key in both places.

Reply:
Yeah, that makes sense to mee